### PR TITLE
[google_maps_flutter_web]: Update instructions to reflect current versions

### PR DESCRIPTION
**Pull request by domesticmouse**
The `google_maps_flutter_web` package `README.md` lists old versions in the usage description. This is confusing to users attempting to use Google Maps in Flutter for web apps.

fixes: https://github.com/flutter/flutter/issues/86631

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Note that unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I [updated pubspec.yaml](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates) with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test exempt.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code

### CODE REVIEW

**Comment by domesticmouse** [comment]
Test exemption request: https://discord.com/channels/608014603317936148/608018585025118217/866467683033808906

**Comment by Hixie** [comment]
test-exempt: only affects documentation

File: packages/google_maps_flutter/google_maps_flutter_web/README.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ This package is not an endorsed implementation of the google_maps_flutter plugin
 
 ```yaml
 dependencies:
-  google_maps_flutter: ^0.5.28
-  google_maps_flutter_web: ^0.1.0
+  google_maps_flutter: ^2.0.0
+  google_maps_flutter_web: ^0.3.0
```
Comment (stuartmorgan) [inline packages/google_maps_flutter/google_maps_flutter_web/README.md:14]: We should just remove this code section, and adjust the line above to say something like: `[...] so you'll need to [add it explicitly](https://pub.dev/packages/google_maps_flutter_web/install)`

We should get out of the business of having doomed-to-bitrot installation examples in our READMEs, and let pub.dev's dynamic page handle that.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/google_maps_flutter/google_maps_flutter_web/README.md (hunk 1 above)
Comment (domesticmouse) [reply packages/google_maps_flutter/google_maps_flutter_web/README.md]: done

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Review by ditman — APPROVED** [review]

**Comment by ditman** [comment]
Very good change, I've just merged it. Thanks!!

**Pull request merged**